src/main/java/frc/robot/subsystems/Leds.java

```diff
@@ -229,4 +269,54 @@ protected void interrupted() {
             }
         };
     }
+
+    public Command turnOnVisionLights() {
+        return new InstantCommand("turn on the color known as vision", this, () -> {
```

Comment:
Could probably make these command names shorter

Reply:
Yes, Darn it rohan